src/Components/ShareCipher/Congrats.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Anime from 'react-anime';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faGraduationCap } from '@fortawesome/free-solid-svg-icons';
+
+
+function Congrats(props) {
+
+    let plaintext = props.cipher[1];               // plaintext of the ciphered message
+    let ciphertext = props.cipher[0];             // current guess at the deciphered message
+    // load the described cipher.
+    
+    return (
+        <div className="container center">
+            <h1 className='title is-size-2 underline mt-4'>Are you the new crack master?</h1>
+
+                <section className='share-section'>
+                    <Anime 
+                    easing="linear"
+                    delay='100' 
+                    opacity={['0%','100%']}
+                    duration="1000"
+                    >
+                    <FontAwesomeIcon icon={faGraduationCap} alt='key' size="6x" color="#FFB800" />
+                        <p className='is-size-3 mb-5'>Congratulations! </p>
+                    </Anime>
```

Comment:
This is rather picky but I'd rather you indent this properly. It helps significantly with readability. If you want to automate this, [there are linters that can do this](https://prettier.io/docs/en/precommit.html)

```suggestion
                <section className='share-section'>
                    <Anime 
                        easing="linear"
                        delay='100' 
                        opacity={['0%','100%']}
                        duration="1000"
                    >
                        <FontAwesomeIcon icon={faGraduationCap} alt='key' size="6x" color="#FFB800" />
                        <p className='is-size-3 mb-5'>Congratulations! </p>
                    </Anime>
```